requirements.txt

```diff
@@ -10,6 +10,5 @@ statsmodels
 seaborn==0.8.1
 matplotlib==2.0.0
 python-dotenv
-cairocffi
```

Comment:
Sure we don't need this?

Reply:
That removal was a mistake, I've fixed it.